specification/monitor/resource-manager/Microsoft.Insights/stable/2018-04-16/scheduledQueryRule_API.json

```diff
@@ -120,7 +120,7 @@
           "default": {
             "description": "Error response describing why the operation failed.",
             "schema": {
-              "$ref": "#/definitions/ErrorResponse"
+              "$ref": "#/definitions/ErrorContract"
```

Comment:
This is a fix to s360 violation :
https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=60BD29A9-717D-4319-A4F4-6DD8D39EE288